components/push/storage/src/db.rs

```diff
@@ -171,6 +175,24 @@ impl Storage for PushDb {
         )?;
         Ok(affected_rows == 1)
     }
+
+    fn get_meta(&self, key: &str) -> Result<Option<String>> {
+        // Get the most recent UAID (which should be the same value across all records,
+        // but paranoia)
+        let query = "SELECT value FROM meta_data where key = :key limit 1";
+        let mut statement = self.prepare(query)?;
+        let mut rows = statement.query_named(&[(":key", &key)])?;
```

Comment:
If you `use sql_support::ConnExt` this would be a lot shorter (something like `self.try_query_one(query, &[(":key", &key)], true)`) and would report query plans if built with --all-features.